### PR TITLE
feat: created avatar and badge components

### PR DESCRIPTION
**Pull request by kedark152**
### feat: Created Avatar and Badge Components.
1. avatar component css code is written in avatar.css file.
2. badge component css code is written in badge.css file.
3. both components are imported in docs.css file.
4. docs.css fle consists of styling specific to docs page.
5. html code for all the components in written in docs.html file.

Please suggest your valuable inputs for merging this branch to dev.
**Screenshots:** 
1. Avatar Component
image

2. Badge Component
image

### CODE REVIEW

File: stylesheets/utilities/variables.css [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    --radius-lg: 15px;
+    --radius-rounded: 999px;
+
+    --wgt-light: 300;
+    --wgt-regular: 400;
+    --wgt-bold-semi: 600;
+    --wgt-bold: 700;
+    --wgt-bold-dark: 900;
+
+    --size-0: 0;
```
Comment (FarhanMobashir) [inline stylesheets/utilities/variables.css]: i think having these many values as css variable doesn't make much sense and also i don't see any pattern in the variable values . you can think of a better approach to this.

**Review by FarhanMobashir — CHANGES_REQUESTED** [review]
too many variable for the size.

**Review by Pratik1005 — APPROVED** [review]
Looks good 👍

File: stylesheets/utilities/variables.css (hunk 1 above)
Comment (kedark152) [reply stylesheets/utilities/variables.css]: Thanks Farhan, will try to update changes.

**Review by FarhanMobashir — COMMENTED** [review]
all good

**Pull request merged**